### PR TITLE
final beta1 fixes

### PR DESCRIPTION
**Pull request by kalefranz**
One fix for python argparse internals calling sys.exit() and another fix for flake8.

### CODE REVIEW

File: conda/cli/main.py [hunk 1]
```diff
@@ -128,7 +128,11 @@ def _main(*args):
         args = args + ('-h',)
 
     p = generate_parser()
-    args = p.parse_args(args[1:])
+    try:
+        args = p.parse_args(args[1:])
+    except Exception as e:
+        import pdb; pdb.set_trace()
+        assert 0
```
Comment (goanpeca) [inline conda/cli/main.py]: Why is this needed?

Comment (kalefranz) [inline conda/cli/main.py]: Why is what needed? 😜

Comment (kalefranz) [inline conda/cli/main.py]: It's gone, completely.  Force pushed.  Just snuck in.
